### PR TITLE
Surgeon/Research/Medimerc borg fixes.

### LINKED ISSUE #4749: Surgeon/Crisis cyborgs can't hold phoron.
#### Brief description of the issue
The medical cyborgs, though supposed to be capable of performing chemistry, can't load phoron into the blender.
#### What you expected to happen
To be able to pick up the phoron with, say, the medical gripper.
#### What actually happened
Could not pick up phoron with any item.
#### Steps to reproduce
Attempt to pick up phoron with, say, the medical gripper.

### LINKED ISSUE #4750: Surgeon and Research borgs can't repair their syringes.
#### Brief description of the issue
Syringes are broken forever if you stab someone with them as a surgeon/research borg. Crisis borgs are fine.
#### What you expected to happen
For the syringe to repair itself when recharging.
#### What actually happened
Broken syringe forever.
#### Steps to reproduce
Be surgeon/research borg. Stab someone with syringe. Cry.

#### Additional info:
Fix going up soon.

### PR DESCRIPTION
**Pull request by mistyLuminescence**
Fixes #4749, fixes #4750 .

Medical grippers can now hold stacks of phoron and medical stacks.
Like Crisis cyborgs, Surgeon, Research and Merc Combat Medic cyborgs can now repair their syringes, if broken, by stepping into a recharge station.
Like Crisis cyborgs, Surgeon cyborgs are now able to recharge their polyacid spray by stepping into a research station.

bless me travis

### CODE REVIEW

File: code/modules/mob/living/silicon/robot/drone/drone_items.dm [hunk 1]
```diff
@@ -58,7 +58,9 @@
 		/obj/item/weapon/reagent_containers/glass,
 		/obj/item/weapon/storage/pill_bottle,
 		/obj/item/weapon/reagent_containers/pill,
-		/obj/item/weapon/reagent_containers/blood
+		/obj/item/weapon/reagent_containers/blood,
+		/obj/item/stack/material/phoron,
+		/obj/item/stack/medical
```
Comment (Anewbe) [inline code/modules/mob/living/silicon/robot/drone/drone_items.dm]: They don't need to hold medical stacks.

Comment (mistyLuminescence) [reply code/modules/mob/living/silicon/robot/drone/drone_items.dm]: Is there a reason why medical grippers held by medical borgs shouldn't be able to pick up common medical items? Happy to remove it, just wondering why.

Comment (Anewbe) [reply code/modules/mob/living/silicon/robot/drone/drone_items.dm]: Borgs, with some exceptions, have the tools they need. Giving them the ability to pick up anything else defeats the purpose of grippers.

**Comment by PrismaticGynoid** [comment]
Would giving crisis borgs chemistry bags be outside the scope of this? Having to transfer ever single bottle and pill from the chemmaster to the fridge gets tedious fast.

**Comment by Anewbe** [comment]
The chem bags were very much out of scope. This is named as a fix. Don't add features to posted bugfix PRs, please.

**Comment by mistyLuminescence** [comment]
Chem bags removed. Worth making another PR for?

**Comment by Anewbe** [comment]
If you'd like to propose chem bags for medical borgs, another PR would be required, yes.

**Pull request merged**